semantic-release/action.yaml

```diff
@@ -24,6 +24,15 @@ inputs:
   semantic-version:
     required: false
     description: "Specify what version of semantic release to use"
+  override-github-ref:
```

Comment:
Add the inputs to the readme as well

Reply:
Or maybe we can start using something like https://github.com/npalm/action-docs/tree/main